Lib/test/libregrtest/main.py

```diff
@@ -37,6 +37,8 @@
     TEMPDIR = tempfile.gettempdir()
 TEMPDIR = os.path.abspath(TEMPDIR)
 
+# Add an auditing hook for all tests to ensure these paths are tested
+sys.addaudithook(lambda *a: None)
```

Comment:
I think this should be guarded with ``if hasattr(sys, 'addaudithook')`` like in ``test_audit.py``

Reply:
Yeah, probably. Just testing stuff right now to see whether it's worth having this on permanently

Reply:
Given the amount of stuff this is flushing out (I should have done it a while ago...), I'm going to leave it in. Hopefully it doesn't hurt the speed of the test runs too much...